howto/monitor/landing.md

```diff
@@ -1,3 +1,12 @@
-The guides in this section describe how to monitor your Anbox Cloud or Anbox Cloud Appliance installation.
+Anbox Cloud gathers various performance metrics and makes them accessible through API endpoints. While Anbox Cloud does not provide its own observability solution, it supports implementing and integrating custom solutions for monitoring performance.
```

Comment:
We do technically provide a dashboard, however it may not fit every users needs

Reply:
What dashboard do you mean? How do you access it?

Reply:
Lets not call out that the charms export a dashboard because it is Grafana-only and makes us again specific to a particular solution. We should keep the landing page as generic as possible